units/crons.go

```diff
@@ -78,5 +79,32 @@ func StartCrons(ctx context.Context, env cedar.Environment) error {
 		return queue.Put(job)
 	})
 
+	if rpcTLS {
+		amboy.IntervalQueueOperation(ctx, remote, 24*time.Hour, time.Now(), opts, func(queue amboy.Queue) error {
+			return queue.Put(NewServerCertRotationJob())
+		})
+		amboy.IntervalQueueOperation(ctx, local, time.Hour, time.Now(), opts, func(queue amboy.Queue) error {
```

Comment:
I'd write a job to do this, and call Close on the environment which you could rig to cancel to starting context in a way...

Reply:
also round now to the top of the hour or something... because we want it fuzzed to a minute and not have it happen too often when we're deploying or the app restarts/crashes.)